scripts/openshift-tools-scripts.spec

```diff
@@ -83,6 +83,7 @@ cp -p monitoring/cron-send-elb-status.py %{buildroot}/usr/bin/cron-send-elb-stat
 cp -p monitoring/ops-ec2-check-tags.py %{buildroot}/usr/bin/ops-ec2-check-tags
 cp -p monitoring/ops-gcp-check-tags.py %{buildroot}/usr/bin/ops-gcp-check-tags
 cp -p monitoring/cron-send-zabbix-too-old.py %{buildroot}/usr/bin/cron-send-zabbix-too-old
+cp -p monitoring/cron-send-router-reload-time.py %{buildroot}/usr/bin/cron-send-router-reload-time
```

Comment:
where is the accompanying piece of the spec file where one of the sub-packages claims /usr/bin/cron-send-router-reload-time?